code/tests/test_helper.brs

```diff
@@ -48,3 +48,10 @@ function isEmptyOrInvalidString(str as dynamic) as boolean
 
     return false
 end function
+
+
+' ************************ Wait Timer Helper ************************
+function _adb_waitFor(time = 1 as integer)
```

Comment:
We should remove it since it appears to be unused.

Reply:
Yes